platform/broadcom/sonic-platform-modules-dell/s6100/scripts/s6100_serial_getty_monitor

```diff
@@ -0,0 +1,4 @@
+#Dell S6100 serial getty monitor
```

Comment:
what about serial-getty@tty**S0**.service?

Reply:
In Dell S6100, only ttyS1 will be active.